cradle-core/src/main/java/com/exactpro/cradle/utils/BookPagesNamesChecker.java

```diff
@@ -21,7 +21,7 @@
 
 public class BookPagesNamesChecker {
 
-    private static final Pattern BOOK_NAME_PATTERN = Pattern.compile("^[a-zA-Z0-9][a-zA-Z0-9_]{0,47}$");
+    public static final Pattern BOOK_NAME_PATTERN = Pattern.compile("[a-zA-Z0-9_]{1,47}$");
```

Comment:
not 47 then. Keyspace name can contain up to 48 symbols. Since we add `book_` before name, it should be decreased to 43